locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
+    "Antarctica": "Antartika",
+    "Antigua And Barbuda": "Antigua ve Barbuda",
     "API Token": "API Jeton",
     "API Token Permissions": "API Jeton İzinleri",
     "API Tokens": "API Jetonları",
     "API tokens allow third-party services to authenticate with our application on your behalf.": "API jetonları, üçüncü taraf hizmetlerin sizin adınıza uygulamamızla kimlik doğrulaması yapmasına izin verir.",
-    "April": "April",
-    "Are you sure you want to delete the selected resources?": "Are you sure you want to delete the selected resources?",
-    "Are you sure you want to delete this file?": "Are you sure you want to delete this file?",
-    "Are you sure you want to delete this resource?": "Are you sure you want to delete this resource?",
+    "April": "Nisan",
+    "Are you sure you want to delete the selected resources?": "Seçili kaynakları silmek istediğinizden emin misiniz?",
+    "Are you sure you want to delete this file?": "Bu dosyayı silmek istediğinizden emin misiniz?",
+    "Are you sure you want to delete this resource?": "Bu kaynağı silmek istediğinizden emin misiniz?",
     "Are you sure you want to delete this team? Once a team is deleted, all of its resources and data will be permanently deleted.": "Bu takımı silmek istediğinizden emin misiniz? Bir ekip silindiğinde, tüm kaynakları ve verileri kalıcı olarak silinecektir.",
     "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Hesabınızı silmek istediğinizden emin misiniz? Hesabınız silindiğinde, tüm kaynakları ve verileri kalıcı olarak silinecektir. Lütfen hesabınızı kalıcı olarak silmek istediğinizi onaylamak için şifrenizi girin.",
-    "Are you sure you want to detach the selected resources?": "Are you sure you want to detach the selected resources?",
-    "Are you sure you want to detach this resource?": "Are you sure you want to detach this resource?",
-    "Are you sure you want to force delete the selected resources?": "Are you sure you want to force delete the selected resources?",
-    "Are you sure you want to force delete this resource?": "Are you sure you want to force delete this resource?",
-    "Are you sure you want to restore the selected resources?": "Are you sure you want to restore the selected resources?",
-    "Are you sure you want to restore this resource?": "Are you sure you want to restore this resource?",
-    "Are you sure you want to run this action?": "Are you sure you want to run this action?",
+    "Are you sure you want to detach the selected resources?": "Seçilen kaynakları ayırmak istediğinizden emin misiniz?",
+    "Are you sure you want to detach this resource?": "Bu kaynağı ayırmak istediğinizden emin misiniz?",
+    "Are you sure you want to force delete the selected resources?": "Seçili kaynakları silmek zorlamak istediğinizden emin misiniz?",
```

Comment:
`"Are you sure you want to force delete the selected resources?": "Seçili kaynakları silmeye zorlamak istediğinizden emin misiniz?",`